apps/meteor/ee/server/apps/marketplace/fetchMarketplaceApps.ts

```diff
@@ -66,20 +66,20 @@ const fetchMarketplaceAppsSchema = z.array(
 		addonId: z.string().optional(),
 		isEnterpriseOnly: z.boolean(),
 		isBundle: z.boolean(),
-		bundedAppIds: z.array(z.string()),
+		bundedAppIds: z.array(z.string()).optional(),
```

Comment:
Curious: shouldn't this be `bundled`?

Reply:
I'll take a look, but yea pretty sure it should